paimon-core/src/test/java/org/apache/paimon/casting/CastExecutorTest.java

```diff
@@ -144,6 +144,42 @@ public void testDecimalToNumeric() {
                 1.23D);
     }
 
+    @Test
+    public void testNumericToString() {
```

Comment:
Cast result should be `BinaryString`, not normal `String`. See `testStringToString`.

Reply:
done.